pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -0,0 +1,434 @@
+                      not
+                      change existing groups; create new groups only)</label>
+                  </div>
+                  <div class="d-flex justify-content-end">
+                    <div class="form-group mb-0">
+                      <input type="hidden" name="__action" value="auto_assessment_groups">
+                      <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                      <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                      <button type="submit" class="btn btn-primary">Group</button>
+                    </div>
+                  </div>
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="modal fade" id="copyAssessmentGroupsModal" tabindex="-1" role="dialog"
+          aria-labelledby="copyAssessmentGroupsModalLabel">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h1 class="h4 modal-title" id="copyAssessmentGroupsModalLabel">Copy group setting</h1>
+              </div>
+              <div class="modal-body">
+                <form name="copy-assessment-group-form" method="POST" enctype="multipart/form-data">
+                  <div class="input-group mb-3">
+                    <select class="custom-select" name="inputGroupSelect01">
+                      <option selected>Choose Assessment...</option>
+                      <% assessment_list_rows.forEach(function(row, iRow) {%>
+                      <option value=<%= row.id %>><%= row.tid %>: <%= row.title %></option>
+                      <% }); %>
+                    </select>
+                  </div>
+                  <div class="d-flex justify-content-end">
+                    <div class="form-group mb-0">
+                      <input type="hidden" name="__action" value="copy_assessment_groups">
+                      <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                      <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                      <button type="submit" class="btn btn-primary">Copy</button>
+                    </div>
+                  </div>
+              </div>
+              </form>
+            </div>
+          </div>
+        </div>
+        <div class="modal fade" id="addGroupModal" tabindex="-1" role="dialog" aria-labelledby="addGroupModalLabel">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h1 class="h4 modal-title" id="addGroupModalLabel">Add a group</h1>
+              </div>
+              <form name="add-Group-form" method="POST">
+                <div class="modal-body">
+                  <div class="form-name">
```

Comment:
I don't believe this class has any meaning.

Separately, consider using bootstrap's `form-group` class here instead.